### PR TITLE
Update SRM to 1.3.0-rc3-24005-00, SCI to 1.2.0-rc3-24005-00 and DSR to 1.0.8-rc2-60325

### PR DESCRIPTION
**Pull request by tmat**
Includes workaround for https://github.com/dotnet/roslyn/issues/10309

### CODE REVIEW

**Comment by tmat** [comment]
@dotnet-bot retest prtest/lin/dbg/unit32 please
// Previous failure: http://dotnet-ci.cloudapp.net/job/roslyn_prtest_lin_dbg_unit32/5887/
// Retest reason:

File: src/Compilers/CSharp/Test/Performance/project.json [hunk 1]
```diff
@@ -1,7 +1,7 @@
 ﻿{
   "dependencies": {
     "Microsoft.DotNet.xunit.performance": "1.0.0-alpha-build0023",
-    "System.IO.FileSystem": "4.0.0"
+    "System.IO.FileSystem": "4.0.1-rc3-24005-00"
```
Comment (agocke) [inline src/Compilers/CSharp/Test/Performance/project.json]: Should stay at RTM

File: src/Scripting/VisualBasic/project.json [hunk 2]
```diff
@@ -1,7 +1,6 @@
 ﻿{
   "dependencies": {
-    "Microsoft.VisualBasic": "10.0.0",
-    "System.AppContext": "4.0.0"
+    "Microsoft.VisualBasic": "10.0.1-rc3-24005-00"
```
Comment (agocke) [inline src/Scripting/VisualBasic/project.json]: This needs to stay at RTM

File: src/Scripting/VisualBasicTest/project.json [hunk 3]
```diff
@@ -1,5 +1,7 @@
 ﻿{
-  "dependencies": { },
+  "dependencies": {
+    "System.AppContext": "4.1.0-rc3-24005-00"
```
Comment (agocke) [inline src/Scripting/VisualBasicTest/project.json]: Same here

File: src/Scripting/CSharpTest/project.json [hunk 4]
```diff
@@ -1,5 +1,7 @@
 ﻿{
-  "dependencies": { },
+  "dependencies": {
+    "System.AppContext": "4.1.0-rc3-24005-00"
```
Comment (agocke) [inline src/Scripting/CSharpTest/project.json]: RTM or nothing

File: src/Scripting/CSharp/project.json [hunk 5]
```diff
@@ -1,8 +1,7 @@
 ﻿{
   "dependencies": {
-    "Microsoft.CSharp": "4.0.0",
-    "System.AppContext": "4.0.0",
-    "System.Dynamic.Runtime": "4.0.10"
+    "Microsoft.CSharp": "4.0.1-rc3-24005-00",
```
Comment (agocke) [inline src/Scripting/CSharp/project.json]: Must stay at RTM

File: src/Compilers/VisualBasic/Test/Performance/project.json [hunk 6]
```diff
@@ -1,7 +1,7 @@
 ﻿{
   "dependencies": {
     "Microsoft.DotNet.xunit.performance": "1.0.0-alpha-build0023",
-    "System.IO.FileSystem": "4.0.0"
+    "System.IO.FileSystem": "4.0.1-rc3-24005-00"
```
Comment (agocke) [inline src/Compilers/VisualBasic/Test/Performance/project.json]: Must be RTM

File: src/Debugging/Microsoft.DiaSymReader.PortablePdb.Tests/project.json [hunk 7]
```diff
@@ -1,12 +1,12 @@
 ﻿{
   "dependencies": {
-    "Microsoft.NETCore.Portable.Compatibility": "1.0.0",
-    "System.Collections": "4.0.10",
-    "System.Diagnostics.Debug": "4.0.10",
-    "System.Runtime": "4.0.20",
-    "System.Runtime.Extensions": "4.0.10",
-    "System.Runtime.InteropServices": "4.0.20",
-    "System.IO.FileSystem": "4.0.0",
+    "Microsoft.NETCore.Portable.Compatibility": "1.0.1-rc3-24005-00",
```
Comment (agocke) [inline src/Debugging/Microsoft.DiaSymReader.PortablePdb.Tests/project.json]: Must be RTM

**Pull request closed without merging**